src/lib.rs

```diff
@@ -1,4 +1,6 @@
 #![no_std]
+#![forbid(unsafe_code)]
+
 //! Provides a macro_rules for making utf-16 literals.
 //!
 //! Outputs are arrays of the correct size. Prefix the macro with `&` to make
```

Comment:
this line of the docs should be updated too.

Reply:
Ah true, will do!